### ISSUE TITLE
Lumen 5.3.3 - Fatal error: Uncaught Error: Call to undefined function GrahamCampbell\Exceptions\Displayers\asset()

### ISSUE THREAD

**Issue by olimorris** [GrahamCampbell/Laravel-Exceptions#95]
In Lumen 5.3.3 I receive the following error when I make a request to a route which is designed to fail:

```
Fatal error: Uncaught Error: Call to undefined function GrahamCampbell\Exceptions\Displayers\asset() in /Users/Oli/Code/MyProject/vendor/graham-campbell/exceptions/src/Displayers/HtmlDisplayer.php:81 Stack trace: #0 /Users/Oli/Code/MyProject/vendor/graham-campbell/exceptions/src/Displayers/HtmlDisplayer.php(67): GrahamCampbell\Exceptions\Displayers\HtmlDisplayer->render(Array) #1 /Users/Oli/Code/MyProject/vendor/graham-campbell/exceptions/src/ExceptionHandlerTrait.php(148): GrahamCampbell\Exceptions\Displayers\HtmlDisplayer->display(Object(Symfony\Component\Debug\Exception\FatalThrowableError), 'e3bd0e5e-3b9c-4...', 500, Array) #2 /Users/Oli/Code/MyProject/vendor/graham-campbell/exceptions/src/ExceptionHandlerTrait.php(88): GrahamCampbell\Exceptions\LumenExceptionHandler->getResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\Debug\Exception\FatalThrowableError), Object(Symfony\Component\Debug\Exception\FatalThrowableError)) #3 /Users/Oli/Code/MyProject/app/Exceptions/Handler.php(48): Gr in /Users/Oli/Code/MyProject/vendor/graham-campbell/exceptions/src/Displayers/HtmlDisplayer.php on line 81

Fatal error: Uncaught Error: Call to undefined function GrahamCampbell\Exceptions\Displayers\asset() in /Users/Oli/Code/MyProject/vendor/graham-campbell/exceptions/src/Displayers/HtmlDisplayer.php:81 Stack trace: #0 /Users/Oli/Code/MyProject/vendor/graham-campbell/exceptions/src/Displayers/HtmlDisplayer.php(67): GrahamCampbell\Exceptions\Displayers\HtmlDisplayer->render(Array) #1 /Users/Oli/Code/MyProject/vendor/graham-campbell/exceptions/src/ExceptionHandlerTrait.php(148): GrahamCampbell\Exceptions\Displayers\HtmlDisplayer->display(Object(Symfony\Component\Debug\Exception\FatalErrorException), 'ac1de7fe-8b4a-4...', 500, Array) #2 /Users/Oli/Code/MyProject/vendor/graham-campbell/exceptions/src/ExceptionHandlerTrait.php(88): GrahamCampbell\Exceptions\LumenExceptionHandler->getResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\Debug\Exception\FatalErrorException), Object(Symfony\Component\Debug\Exception\FatalErrorException)) #3 /Users/Oli/Code/MyProject/app/Exceptions/Handler.php(48): Gr in /Users/Oli/Code/MyProject/vendor/graham-campbell/exceptions/src/Displayers/HtmlDisplayer.php on line 81
```

I fix this by commenting out these two lines on line 81 in `GrahamCampbell\Exceptions\Displayers\HtmlDisplayer.php`:

```
$info['home_url'] = asset('/');
$info['favicon_url'] = asset('favicon.ico');
```

**Comment by GrahamCampbell** [comment]
Thanks for the report. Does https://github.com/GrahamCampbell/Laravel-Exceptions/pull/96 fix this for you?

**Comment by GrahamCampbell** [comment]
@olimorris Should be fixed on v9.3.0.
